src/pages/Profile.jsx

```diff
@@ -89,24 +106,43 @@ const Profile = () => {
 
         <ProfileComponent clientProfile={clientProfile} />
 
-        <Button
-          variant="contained"
-          color="secondary"
-          size="small"
-          aria-label="Add Client Button"
-          startIcon={<AddIcon />}
-          onClick={() => setShowModal(true)}
-        >
-          Add Document
-        </Button>
-        <UploadDocumentModal showModal={showModal} setShowModal={setShowModal} />
-        <DocumentTable />
         {!client && (
           <>
-            <SetAclPermsDocContainerForm />
-            <SetAclPermissionForm />
+            <Container sx={{ display: 'flex', justifyContent: 'center', gap: 2 }}>
+              <Button
+                variant="contained"
+                color="primary"
+                size="small"
+                aria-label="Set Container Permissions Button"
+                startIcon={<SettingsIcon />}
+                onClick={() => setShowAclPermsDocContainerModal(true)}
+              >
+                Container Permissions
```

Comment:
Think we can rephrase this to use something else other than `Container Permissions`. Not really sure what at the moment though.

Reply:
All Document Permissions?

Reply:
Think that sounds alright.

Reply:
Actually, what about "Set Permission for All Documents"?

Reply:
That should be fine.